src/Constants.js

```diff
@@ -53,6 +53,44 @@ export const views = {
   }
 };
 
+export const descriptions = {
+  Projection: {
+    TITLE: 'Projection',
+    CONTENTS: [
+      projectionStill,
+      'Improve your projection skill!',
+      '1. Click MIC button',
+      "2. Say something while looking dB level. Once you projection's dB is over Proper Level, try to keep it up.",
```

Comment:
Maybe...
Practice speaking while looking at the dB level. Once your voice is over the Proper Level, try to keep it up